### PR TITLE
Fix references to `methods` attr

### PR DESCRIPTION
**Pull request by akulakov**

### CODE REVIEW

File: Doc/library/crypt.rst [hunk 1]
```diff
@@ -93,7 +93,7 @@ The :mod:`crypt` module defines the following functions:
    may be available on all platforms), or a full encrypted password
    including salt, as returned by this function.  If *salt* is not
    provided, the strongest method will be used (as returned by
```
Comment (nanjekyejoannah) [inline Doc/library/crypt.rst]: If `methods` is not a function, do we need the wording `as returned by` ? We usually use return for results of functions and methods. Am assuming due diligence, that `methods` is actually an attribute, not a method, therefore someone else can confirm on this.

**Comment by akulakov** [comment]
Yep, methods is just a simple python list in fact, I will update the
wording..

**Comment by akulakov** [comment]
@nanjekyejoannah please see https://github.com/python/cpython/blob/main/Lib/crypt.py#L86

File: Doc/library/crypt.rst [hunk 2]
```diff
@@ -92,8 +92,8 @@ The :mod:`crypt` module defines the following functions:
    :func:`mksalt`, one of the ``crypt.METHOD_*`` values (though not all
    may be available on all platforms), or a full encrypted password
    including salt, as returned by this function.  If *salt* is not
-   provided, the strongest method will be used (as returned by
-   :func:`methods`).
+   provided, the strongest method available in :attr:`methods` will be used.
+
```
Comment (JelleZijlstra) [inline Doc/library/crypt.rst]:
```suggestion
```
Unnecessary extra line

**Review by JelleZijlstra — APPROVED** [review]
Looks good

File: Doc/library/crypt.rst [hunk 3]
```diff
@@ -125,8 +124,8 @@ The :mod:`crypt` module defines the following functions:
 .. function:: mksalt(method=None, *, rounds=None)
 
    Return a randomly generated salt of the specified method.  If no
-   *method* is given, the strongest method available as returned by
-   :func:`methods` is used.
+   *method* is given, the strongest method available in :attr:`methods` is
```
Comment (JelleZijlstra) [inline Doc/library/crypt.rst]:
```suggestion
   *method* is given, the strongest method available in :attr:`methods` is
```

**Review by JelleZijlstra — APPROVED** [review]

**Comment by akulakov** [comment]
@JelleZijlstra thanks for reviewing!

**Pull request merged**